docs/api.md

```diff
@@ -321,6 +323,32 @@ A stream of JSON objects. When finished, `status` is `success`.
 }
 ```
 
+### Create a Blob
```

Comment:
We should probably add another section showing `HOST /api/blobs:digest` to "Check if Blob Exists".